<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python coating control v1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>DCM controls</string>
    </property>
    <widget class="QRadioButton" name="Button_DCM_ON">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DCM ON</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Button_DCM_OFF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DCM OFF</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Jogg/Step position adjustments</string>
    </property>
    <widget class="QPushButton" name="Step_move_up_screen_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Up Screen (8)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Step_move_down_screen_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Down Screen (2)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Step_move_forward_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward(6)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Step_move_back_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Back(4)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Step_move_up_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Up /\</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Step_move_down_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Down\/</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters of the move</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>81</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Step size</string>
     </property>
     <widget class="QRadioButton" name="radioButton_1000um">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>16</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1000 um</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_200um">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>33</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>200 um</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_50um">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>49</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>50 um</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_20um">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>66</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>20 um</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="JoggRadioButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Jogg</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="StepRadioButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>81</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Speed</string>
     </property>
     <widget class="QRadioButton" name="HighSpeedRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>High</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="MediumSpeedRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="LowSpeedRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Low</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Step_move_back_3_cm_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back 3 cm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Step_move_forward_2_9_cm_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward 2.9 cm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="This_is_n1_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>This Is # 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Go_to_n1_Button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to # 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_needle_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current needle: </string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>FileDialog</string>
    </property>
    <widget class="QPushButton" name="File_Dialog_Button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OPEN FILE</string>
     </property>
    </widget>
    <widget class="QLabel" name="code_file_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No running code file is chosen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Number of coats</string>
    </property>
    <widget class="QDoubleSpinBox" name="Number_of_coats_SpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>25.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Download_current_coordinates_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download current coordinates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File_Run_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RUN FILE</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="current_cycle_value">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Coat cycle: </string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopMotionButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Running time, s</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeRunningSeconds">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Align every</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Check correction</string>
      </property>
      <widget class="QPushButton" name="Accept_and_Go_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Accept and go to Next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Go_to_next_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Go to Next</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Align n</string>
     </attribute>
     <widget class="QPushButton" name="ButtonAlignNAlignAndGoToNext">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Align and go to next</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNextToAlign">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Next to align: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonAlignNGoBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GoBack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonAlignNGoForward">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GoForward</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonCalculateAlignment">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Alignment</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelJustAligned">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Just aligned: </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Save(d)</string>
     </attribute>
     <widget class="QPushButton" name="DownloadAlignmentButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Download alignment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveAlignmentButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save alignment</string>
      </property>
     </widget>
     <widget class="QLabel" name="alignment_file_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>No alignment file</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDefaultAlignment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>142</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Default alignment</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
